Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -465,10 +486,24 @@ extension OrderStore {
 // MARK: - Storage: Search Results
 //
 private extension OrderStore {
+    /// Updates the specified ReadOnly Order Entity.
+    ///
+    /// - Returns: The updated order, prior to performing the update operation.
+    ///
+    @discardableResult
+    func updateStoredOrder(readOnlyOrder: Networking.Order) -> Networking.Order {
+        guard let storageOrder = storageManager.viewStorage.loadOrder(siteID: readOnlyOrder.siteID, orderID: readOnlyOrder.orderID) else {
```

Comment:
We only update existing orders. IMO, there is no point in optimistically updating a non-existent order 🙂